eZ/Bundle/EzPublishCoreBundle/Tests/DependencyInjection/Configuration/SiteAccessAware/ContextualizerTest.php

```diff
@@ -203,6 +228,10 @@ public function testMapConfigArray()
             $expectedMergedSettings['sa3'],
             $containerBuilder->getParameter("$this->namespace.sa3.foo_setting")
         );
+        $this->assertSame(
+            $expectedMergedSettings['sa_group1'],
+            $containerBuilder->getParameter("$this->namespace.sa_group1.foo_setting")
```

Comment:
Is this interpolation needed here (and in the below changes)?
```suggestion
            $containerBuilder->getParameter($this->namespace.sa_group1.foo_setting)
```